scripts/inference.py

```diff
@@ -190,7 +190,7 @@ def print_result(result):
         return
     # stop at EOS token if present
     result = generation.truncate_after_eos(
-        result, tokenizer.convert_tokens_to_ids("</s>")
+        result, tokenizer.convert_tokens_to_ids('<\s>')
```

Comment:
the linter is wanting this to be double-quotes. also though we probably want to change it to tokenizer.eos_token_id or wahtever it's called

Reply:
tokenizer.eos_token_id is an integer, so I am not able to directly replace it for '<\s>'. is str(tokenizer.eos_token_id) the same thing as the string? it compiles but I'm not sure if they are the same.

Reply:
we just change `tokenizer.convert_tokens_to_ids("</s>")` to `tokenizer.eos_token_id`, the convert function returns an int